src/logging_unittest.cc

```diff
@@ -251,8 +251,8 @@ void TestLogging(bool check_counts) {
     PLOG_EVERY_N(ERROR, 2) << "Plog every 2, iteration " << COUNTER;
     errno = old_errno;
 
-    LOG_EVERY_N(ERROR, 3) << "Log every 3, iteration " << COUNTER << endl;
-    LOG_EVERY_N(ERROR, 4) << "Log every 4, iteration " << COUNTER << endl;
+    //LOG_EVERY_N(ERROR, 3) << "Log every 3, iteration " << COUNTER << endl;
+    //LOG_EVERY_N(ERROR, 4) << "Log every 4, iteration " << COUNTER << endl;
```

Comment:
Maybe using `__attribute__((unused))` can help? The typedef is likely generated using macros. Try searching for a small part of typedef's name.